src/app/modules/item/components/item-dependencies/item-dependencies.component.ts

```diff
@@ -37,7 +43,13 @@ export class ItemDependenciesComponent implements OnChanges, OnDestroy {
     data ? this.op?.toggle(data.event, data.target) : this.op?.hide();
   });
 
-  constructor(private getItemPrerequisitesService: GetItemPrerequisitesService) {
+  createInProgress$ = new BehaviorSubject<boolean>(false);
```

Comment:
Here when looking at the usage of this subject, I think a boolean instead of a subject is enough

Reply:
Fixed